File: backend/src/models/TLUUTAMCHITIET.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class TLUUTAMCHITIET extends Model {
    public id!: any;
    public note!: any;
    public tluutamid!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public usercreatedid!: any;
    public soluong!: any;
    public dongia!: any;
    public thanhtien!: any;
    public tilegiamgia!: any;
    public baohanh!: any;
    public dmathangid!: any;
    public soluongchuaquydoi!: any;
    public ddonvitinhid!: any;
}

TLUUTAMCHITIET.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    TLUUTAMID: {
        type: DataTypes.INTEGER,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    SOLUONG: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    DONGIA: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    THANHTIEN: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    TILEGIAMGIA: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    BAOHANH: {
        type: DataTypes.STRING,
        
    },
    DMATHANGID: {
        type: DataTypes.INTEGER,
        
    },
    SOLUONGCHUAQUYDOI: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    DDONVITINHID: {
        type: DataTypes.INTEGER,
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'TLUUTAMCHITIET',
    tableName: 'TLUUTAMCHITIET',
    timestamps: false
});

export default TLUUTAMCHITIET;

File: frontend/Models/TLUUTAMCHITIET.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class TLUUTAMCHITIET
    {
        public int? Id { get; set; }
        public string Note { get; set; }
        public int? TluutamId { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? UsercreatedId { get; set; }
        public decimal? Soluong { get; set; }
        public decimal? Dongia { get; set; }
        public decimal? Thanhtien { get; set; }
        public decimal? Tilegiamgia { get; set; }
        public string Baohanh { get; set; }
        public int? DmathangId { get; set; }
        public decimal? Soluongchuaquydoi { get; set; }
        public int? DdonvitinhId { get; set; }
    }
}
